### PR TITLE
pixtral notebook

### PR DESCRIPTION
**Pull request by eaidova**

### CODE REVIEW

**Comment by eaidova** [comment]
<p>that should be obvious as 12billion parameters, but ok, I'll ad</p>

---

 <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2426/discussion/#comment-2388236028'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

File: notebooks/pixtral/gradio_helper.py [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+            yield buffer
+
+    demo = gr.ChatInterface(
+        fn=bot_streaming,
+        title=f"{model_name} with OpenVINO",
+        examples=[
+            {"text": "What is the text saying?", "files": ["./small.png"]},
+            {"text": "What does the chart display?", "files": ["./chart.png"]},
+        ],
+        description=f"{model_name} with OpenVINO. Upload an image and start chatting about it, or simply try one of the examples below. If you won't upload an image, you will receive an error.",
```
Comment (aleksandr-mokrov) [inline notebooks/pixtral/gradio_helper.py:115]: I receive an error `TypeError: can only concatenate list (not "str") to list` if I continue chatting after first message. It doesn't matter what's in the second message, just a message or a message with a picture.

Comment (eaidova) [reply notebooks/pixtral/gradio_helper.py:115]: fixed

File: notebooks/pixtral/gradio_helper.py [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+        stop_btn="Stop Generation",
```
Comment (aleksandr-mokrov) [inline notebooks/pixtral/gradio_helper.py]: After pushing "Stop button" and sending a new request or retrying an old request an error raises `RuntimeError: Infer Request is busy`.

Another one error:
1. Send a picture and message.
2. After response send another one text request.
3. Press "Undo" two times and send request. In this case the bounded image and message will be unbounded that causes error:
`jinja2.exceptions.TemplateError: After the optional system message, conversation roles must alternate user/assistant/user/assistant/...`

Comment (eaidova) [reply notebooks/pixtral/gradio_helper.py]: request busy, it is runtime error sometimes it happens when it is not enough capacity to process several requests simulteniously

Comment (eaidova) [reply notebooks/pixtral/gradio_helper.py]: another problem is model chat template limitation, clicking undo twice, you change chain of messages (user-assistant-user-assistant -> user-user) that is not supported by model, so for avoid confusion I disable undo and retry buttons

**Review by aleksandr-mokrov — APPROVED** [review]

**Pull request merged**